src/commands/flex/plugins/start.ts

```diff
@@ -1,15 +1,76 @@
+import { join } from 'path';
+
+import { flags } from '@oclif/command';
+
 import { createDescription } from '../../../utils/general';
-import FlexPlugin from '../../../sub-commands/flex-plugin';
+import FlexPlugin, { ConfigData, SecureStorage } from '../../../sub-commands/flex-plugin';
+import { filesExist } from '../../../utils/fs';
+
 /**
  * Starts the dev-server for building and iterating on a flex-plugin
  */
 export default class FlexPluginsStart extends FlexPlugin {
-  static description = createDescription('Starts a dev-server to build the Flex plugin locally', true);
+  static description = createDescription('Starts a dev-server to build the Flex plugin locally', false);
+
+  static flags = {
+    ...FlexPlugin.flags,
+    name: flags.string({
+      multiple: true,
+      char: 'n',
+    }),
+    'include-remote': flags.boolean(),
+  };
+
+  constructor(argv: string[], config: ConfigData, secureStorage: SecureStorage) {
+    super(argv, config, secureStorage, {});
+
+    if (this._flags['include-remote'] || this._flags.name) {
+      this.opts.runInDirectory = false;
+    }
+  }
 
   /**
    * @override
    */
   async doRun() {
-    await this.runScript('start');
+    const names = this._flags.name;
+    const includeRemote = this._flags['include-remote'];
+    const flexArgs: string[] = [];
+    const pluginArgs: string[][] = [];
+
+    if (names) {
+      for (const name of names) {
+        flexArgs.push('--name', name);
+        pluginArgs.push(['--name', name]);
+      }
+    }
+
+    if (includeRemote) {
+      flexArgs.push('--include-remote');
+    }
+
+    // If running in a plugin directory, append it to the names
+    if (filesExist(join(this.cwd, 'public', 'appConfig.js')) && !flexArgs.includes(this.pkg.name)) {
```

Comment:
There is already a method `this.isPluginsDirectory()` use that.

Reply:
I was using that method previously but it doesn't work correctly as it returns true if !this.opts.runInDirectory, so when a --name is provided, the if statement doesn't function as I need it to

Reply:
I'll fix that